.appveyor.yml

```diff
@@ -18,7 +18,7 @@ environment:
     #- vscode: test:insiders-without-system-tests
 
 install:
-  - ps: Install-Product node 8 x64
+  - ps: Install-Product node 10.2.0 x64
   - appveyor-retry npm install
```

Comment:
I'm thinking about updating this as well cause appveyor is installing version 5.6.0 of npm but would first like to get some feedback about the version changes.